docs/pyretriever.rst

```diff
@@ -134,7 +131,7 @@ There are in all 4 default parameters.
 Install Datasets
 ================
 
-Retriever supports scripts installation into 7 major formats or engines as we call them.
+Retriever supports the installation of datasets into 7 major databases and file formats.
```

Comment:
`Retriever supports installation of datasets into 7 major databases and file formats.` Extra `the`.